lib/oidcstrategy.js

```diff
@@ -143,6 +143,22 @@ function Strategy(options, verify) {
       This is fine if you are expecting multiple organizations to connect to your app.
       Otherwise you should validate the issuer.`);
   }
+
+  // validate other necessary option items provided, we validate them here and only once
+  var itemsToValidate = objectTransform({
+    source: options,
+    pick: ['clientID', 'callbackURL', 'responseType', 'responseMode']
+  });
+  var validatorConfiguration = {
+    clientID: Validator.isNonEmpty,
+    callbackURL: Validator.isNonEmpty,
+    // callbackURL: Validator.isURL  // We allow http for now
```

Comment:
Well, then isURL is poorly named. :) Is there a validation property that will validate that it's a URL that can be http or https?